prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0, name="Car"):
+        """Initialise a Car instance.
```

Comment:
Question? For doc-string multiple lines, I thought the quotations marks on first line  and last lines and text on in between lines??

Reply:
Perfect, Solved.